rlkit/envs/wrappers/dm_backend_wrappers.py

```diff
@@ -505,10 +505,17 @@ def render(
         if mode == "human":
             self.renderer.render_to_window()
         elif mode == "rgb_array":
-            return self.renderer.render_offscreen(
+            # TODO Why reverse iterate?
+            primary_img = self.renderer.render_offscreen(
```

Comment:
can you rename this to fixed view img - probably a bit clearer

Reply:
sure, that makes sense